app/scripts/views/settings/clients.js

```diff
@@ -76,7 +76,7 @@ define(function (require, exports, module) {
           }
         } else {
```

Comment:
Will chat with @ryanfeeley about it, we need to tweak the `t('started syncing ` string instead